### PR TITLE
chore(rust): use more semantic method about judging the file path exists.

### PR DESCRIPTION
**Pull request by wangziling**
Better to use the more semantic method about judging the file path exists.😉.

image

### CODE REVIEW

File: crates/rolldown_binding/src/bundler.rs
Comment (hyf0) [inline crates/rolldown_binding/src/bundler.rs:1]: Could you revert change on this file? This code is for wasm specially.

**Comment by wangziling** [comment]
Hi @hyf0 ~~~ Should I create a new branch to make a better git graph/history without the `rever:` commits?

**Comment by hyf0** [comment]
> Hi @hyf0 ~~~ Should I create a new branch to make a better git graph/history without the `rever:` commits?

No.

**Comment by wangziling** [comment]
> > Hi @hyf0 ~~~ Should I create a new branch to make a better git graph/history without the `rever:` commits?
> 
> No.

Okay~ Thanks~

Sorry about wrongly changing the `wasm32` related codes without fully testing and fully understanding😫.

**Review by hyf0 — APPROVED** [review]
Thanks

**Comment by wangziling** [comment]
> Thanks

You are welcome😉~

That's just a tiny contribution~

**Pull request merged**